plsync/mlabzone.header.in

```diff
@@ -0,0 +1,35 @@
+;
+; Primary DNS zone file
+; NOTE: this file was automatically generated!!!
+; NOTE: DO NOT EDIT
+;
+
+$ORIGIN %(domain)s.
+$TTL    %(ttl)s
+
+@       IN      SOA     %(primary_nameserver)s. %(support_email)s. (
+        %(serial)s      ; Serial
+        %(refresh)s     ; Refresh
+        %(retry)s       ; Retry
+        %(expire)s      ; Expire
+        %(minttl)s )    ; Negative caching TTL
+@       IN      NS      %(primary_nameserver)s.
+@       IN      NS      %(secondary_nameserver)s.
+
+; TODO: fix A and MX records appropriately.
+@       IN      A       128.112.139.90
+@       IN      MX 0    mail.planet-lab.org.
+*       IN      MX 0    mail.planet-lab.org.
+
+; ---------- WWW ----------
+;
+www     IN      A       128.112.139.90
+
+; ---------- DONAR -------
```

Comment:
I can't see any actual harm in having these here, but are these DONAR records necessary any longer? I see that we are running some process named `resolve-by-mlabns.py` on these mlab_utility slivers, but my understanding was that DONAR (NODAR?) has been defunct for a long time now.